script/_common

```diff
@@ -25,13 +25,8 @@ function docker_exec() {
 function docker_bundle_exec() {
   docker_build
 
-  VCR_ARGS=""
-  if [ -n "$VCR" ]; then
-    VCR_ARGS="--env \"VCR=$VCR\""
-  fi
```

Comment:
This code is simply trying to set `--env VCR=$VCR` only when `$VCR` is set... and can be replaced with simply `--env VCR`...